src/views/TechnicalAssistance/Homepage.tsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file "Technical Assistance" (TRB) homepage.
+ * Created with reference to `src/views/MakingARequest`, `src/views/MyRequests/Table`.
+ */
+
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRouteMatch } from 'react-router-dom';
+import {
+  Column,
+  useFilters,
+  useGlobalFilter,
+  usePagination,
+  useSortBy,
+  useTable
+} from 'react-table';
+import { useQuery } from '@apollo/client';
+import { Button, Link, Table } from '@trussworks/react-uswds';
+
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageLoading from 'components/PageLoading';
+import GlobalClientFilter from 'components/TableFilter';
+import TablePageSize from 'components/TablePageSize';
+import TablePagination from 'components/TablePagination';
+import TableResults from 'components/TableResults';
+import GetTrbRequestsQuery from 'queries/GetTrbRequestsQuery';
+import {
+  GetTrbRequests,
+  // eslint-disable-next-line camelcase
+  GetTrbRequests_trbRequestCollection
+} from 'queries/types/GetTrbRequests';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon
+} from 'utils/tableSort';
+import NotFound from 'views/NotFound';
+import { formatDate } from 'views/SystemProfile';
+
+function Homepage() {
+  const { t } = useTranslation('technicalAssistance');
+  const { url } = useRouteMatch();
+
+  const { loading, error, data } = useQuery<GetTrbRequests>(
+    GetTrbRequestsQuery
+  );
+
+  const trbRequests = data?.trbRequestCollection || [];
+
+  // eslint-disable-next-line camelcase
+  const columns = useMemo<Column<GetTrbRequests_trbRequestCollection>[]>(() => {
+    return [
+      {
+        Header: t<string>('table.header.requestName'),
+        accessor: 'name'
+      },
+      {
+        Header: t<string>('table.header.submissionDate'),
+        accessor: 'createdAt',
+        Cell: ({ value }) => formatDate(value)
+      },
```

Comment:
With more seeded data, this will not sort correctly.  Global search and column sorting work on the accessor value, not the cell value.  Honestly I think the custom sort function should be modified to not parse date before passing into the sort function (It's fine to parse it as a date string, not a date object).  The sort function itself can attempt to parse dates to compare values.  The formatting of the date string should happen in the accessor basically. This may sound confusing, but I just did some work on this for MINT, we can pair on explaining how this really works.